code/modules/food/beverages/bottler.dm

```diff
@@ -28,9 +26,8 @@
 			var/datum/bottler_recipe/recipe = new type
 			if(recipe.result) // Ignore recipe subtypes that lack a result
 				available_recipes += recipe
-				acceptable_items |= recipe.obj_1_item
-				acceptable_items |= recipe.obj_2_item
-				acceptable_items |= recipe.obj_3_item
+				for(var/i = 1, i <= recipe.ingredients.len, i++)
```

Comment:
`for(var/i in 1 to recipe.ingredients.len)`

Not only is it faster, but it's more readable.